config/locales/doc_auth/es.yml

```diff
@@ -315,12 +316,16 @@ es:
         proceso.
       header: ¿No está listo para enviar las fotos?
     tips:
-      document_capture_header_text: 'Para obtener los mejores resultados:'
       document_capture_hint: Debe ser un JPG o PNG
       document_capture_id_text1: Use un fondo oscuro
       document_capture_id_text2: Tome la foto en una superficie plana
       document_capture_id_text3: No use el flash de su cámara
       document_capture_id_text4: El tamaño del archivo debe ser de al menos 2 MB
+      document_capture_selfie_id_header_text: Consejos para obtener fotografías nítidas
+      document_capture_selfie_selfie_text: Consejos para obtener una foto nítidas
+      document_capture_selfie_text1: Mantenga el dispositivo al mismo nivel que los ojos
+      document_capture_selfie_text2: Asegúrese de que toda su cara sea visible
```

Comment:
Change "Asegúrese de que toda su cara sea visible" to "Asegúrate de que tu rostro completo sea visible"